octoprint_octorelay/__init__.py

```diff
@@ -313,12 +313,12 @@ def print_started(self):
             relay_pin = int(settings["relay_pin"])
             inverted = settings['inverted_output']
             autoONforPrint = settings['autoONforPrint']
+            cmdON = settings['cmdON']
             active = settings["active"]
             if autoONforPrint and active:
                 self._logger.debug("turning on pin: {}, index: {}".format(relay_pin, index))
                 GPIO.setup(relay_pin, GPIO.OUT)
```

Comment:
```suggestion
```

removing this line here, since:
- same one exists in the `turn_on_pin` which is called in the next line
- for consistency with `turn_off_pin` method doing the same